src/cloudant/result.py

```diff
@@ -503,10 +540,32 @@ class QueryResult(Result):
     and ``limit`` are not permitted to be part of the query callable or be
     included as part of the QueryResult customized parameters.
 
+    List of valid QueryResult customization options:
+    :param str bookmark: A string that enables you to specify which page of
+        results you require. Only valid for queries using indexes of type
+        *text*.
+    :param list fields: A list of fields to be returned by the query.
+    :param int page_size: Sets the page size for result iteration.  Default
+        is 100.
+    :param int r: Read quorum needed for the result.  Each document is read
+        from at least 'r' number of replicas before it is returned in the
+        results.
+    :param str selector: Dictionary object describing criteria used to
+        select documents.
+    :param list sort: A list of fields to sort by.  Optionally the list can
+        contain elements that are single member dictionary structures that
+        specify sort direction.  For example
+        ``sort=['name', {'age': 'desc'}]`` means to sort the query results
+        by the "name" field in ascending order and the "age" field in
+        descending order.
+    :param str use_index: Identifies a specific index for the query to run
+        against, rather than using the Cloudant Query algorithm which finds
+        what it believes to be the best index.
+
+
     :param query: A reference to the query callable that returns
         the JSON content result to be wrapped.
-    :param options: See :func:`~cloudant.query.Query.make_result` for a
-        list of valid query result customization options.
+    :param options: A list of valid QueryResult customization options.
```

Comment:
Apply the same comments that I made on the Result docstring changes here as well.